### DISCUSSION TITLE
How do you get back a response from a RPC function call

### DISCUSSION THREAD

**Discussion by VicSmithVercel** [supabase-community/supabase-csharp#170] [Q&A]
Using the supabase.Rpc I continue to get back: 

Supabase.Postgrest.Responses.BaseResponse

using:

`await supabase.Rpc()`

My function returns a primary key of the newly created record.

How can i get that into:
 `var response =await supabase.Rpc()`
 
 ?

**Comment by alphanumericaracters** [comment]
You get it?
